tests/test_data.py

```diff
@@ -170,16 +171,22 @@ def create_df_8():
     return df
 
 
-def create_df_9():
+def create_df_9(library="pandas"):
```

Comment:
re these test df functions, these look good but we have adjusted the approach a bit in Justin's inbound PR. Might be worth sitting on this until you can pull in changes and match the style there

Reply:
If you merge in main now you should see Justin's changes :)

Reply:
Cool :) Hopefully in line with what we want to do now - [here](https://github.com/lvgig/tubular/pull/337/commits/6c6e161396afacbd77c436c07e631f65f26c2876)